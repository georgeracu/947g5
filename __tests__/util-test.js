'use strict';

import renderer from 'react-test-renderer';
import util from '../utils/util';
import React from 'react';

describe('The utility component', () => {
  test('Renders correctly', () => {
    const tree = renderer.create(<util />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
