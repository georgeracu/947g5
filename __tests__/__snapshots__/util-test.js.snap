// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The utility component Renders correctly 1`] = `<util />`;
